### PR TITLE
Chrome 87 added support for text-underline-offset

### PR DESCRIPTION
**Pull request by stapelberg**
Changelog:
https://chromium.googlesource.com/chromium/src/+log/86.0.4240.198..87.0.4280.66?pretty=fuller&n=10000
Commit:
https://chromium.googlesource.com/chromium/src/+/6ee93f1d7e463530d9423c79b5a58ca2bf881dcb

Tested manually in Chrome 87’s inspector.

A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [x] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

File: css/properties/text-underline-offset.json [hunk 1]
```diff
@@ -6,8 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-underline-offset",
           "support": {
             "chrome": {
-              "version_added": false,
-              "notes": "See Chrome Platform Status entry for <a href='https://chromestatus.com/feature/5747636764147712'>text-decoration-thickness, text-underline-offset and from-font keyword for text-underline-position</a>."
+              "version_added": "87"
             },
             "chrome_android": {
               "version_added": false
```
Comment (ddbeck) [inline css/properties/text-underline-offset.json:12]:
```suggestion
              "version_added": "87"
```

The same would go for `webview_android` below

**Review by ddbeck — CHANGES_REQUESTED** [review]
Hi @stapelberg and welcome to BCD! Thank you for this PR.

I see that the changes you linked to cover several changes at once: `text-decoration-thickness`, `text-underline-offset` and the `from-font` keyword for `text-underline-position`. Would you mind if we expanded the scope to include all of these at once?

Either way, it also appears that Chrome intends to ship this regardless of platform, so we'll need to change the corresponding values for `"chrome_android"` and `"webview_android"` at the same time. I've left a line comment to illustrate.

Thank you!

File: css/properties/text-underline-offset.json (hunk 1 above)
Comment (stapelberg) [reply css/properties/text-underline-offset.json]: Done

**Comment by stapelberg** [comment]
All done!

**Review by ddbeck — APPROVED** [review]
Outstanding. Thank you! 🎉

**Pull request merged**